src/Sulu/Component/Rest/Tests/Unit/RequestParametersTraitTest.php

```diff
@@ -59,99 +54,118 @@ private function getGetBooleanRequestParameterReflection()
 
     public function testGetRequestParameter()
     {
-        $request = $this->getRequestMock(
-            [
-                ['test', null, false, 'data'],
-                ['none', 'default', false, 'default'],
-            ]
-        );
+        $request = $this->prophesize(Request::class);
```

Comment:
Why not simply using the real Request object? That's also what the prophecy team suggests: https://github.com/phpspec/prophecy/issues/86#issuecomment-64948339